psi4/src/psi4/occ/cepa_iterations.cc

```diff
@@ -41,7 +41,7 @@ namespace occwave {
 void OCCWave::cepa_iterations() {
     outfile->Printf("\n  \n");
     outfile->Printf(" ============================================================================== \n");
-    outfile->Printf(" ================ Performing LCCD iterations... =============================== \n");
+    outfile->Printf(" ================ Performing LCCD iterations with REMP modification... =============================== \n");
```

Comment:
Do we always want this to run? What if it is a genuine CEPA computation?

Reply:
afai see, `cepa_iterations` gets run for LCCD and `remp_iterations` for REMP2, so I'm removing the "with REMP modification"